projboard/templates/show/read_article.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+<head>
+    <link rel="stylesheet" type="text/css" href="{% static 'css/read_article.css' %}">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />    
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Anton&family=Zilla+Slab:wght@400;700&display=swap" rel="stylesheet">
+    
+</head>
+<main>
+    <article>
+        <header>
+            <h2>{{article.title}}</h2>
+            <div class="footer">
+                <p class="nickname"> &#128100; {{article.user_id.nickname}} | &#8987; {{article.date}} </p>
+                <p id="category">
+                    <strong>Category</strong>: 
+                    {% if article.subject_id == null %}
+                        Other
+                    {% else %}
+                        {{article.subject_id}}
+                    {% endif %} 
+                </p>
+                <span class="material-symbols-outlined">
+                    reviews
+                </span>
+                <p><strong>
+                    Views: {{ article.num_of_views }}  
+                </strong></p>
+            </div>
+            <hr>
+            <p>{{article.content}}</p>
+        </header>
+        <footer>
+            <hr>
+            <form method="POST">
+            <div class="footer">
+                {% csrf_token %}
+                <button class="material-symbols-outlined" type="submit" >
+                    thumb_up
+                </button>
```

Comment:
@Yarboa , here is the GUI that able user to like/dislike the article , its the same button for both funconality (like & dislike) .

Reply:
Sorry but assume you have few buttons. how do you know which one is submitted? 

               {% csrf_token %}
                <button class="material-symbols-outlined" type="submit">
                    thumb_up
                </button>
                
Are you assuming this is the only POST request parsed to view method?

See Line L71 in the View file, You are not testing like or dislike?
You should have 2 choices for the button, 

This way you can test the post requests for like or dislike an article later on, do you get my point here?

Reply:
i'm gonna give an explaination :
in order to make it simple => it' like or unlike button => so i realy don't think there is a need to another button => when user click on it , i check if that user have already liked the blog , if yes so i'm assuming that he want to delete the like  ,else he want to like it => all that done in the view , so i don't need to distingush between like or unlike.

Now , i think you want me to use DELETE Method , is it right ??
im sorry if i didn't get you point

Reply:
No, 
see this tutorial it is a bit different but the same :)
https://www.youtube.com/watch?v=xqFM6ykQEwo

Reply:
done